src/gui/dialogs/title_screen.cpp

```diff
@@ -162,6 +167,16 @@ static void launch_lua_console()
 	gui2::dialogs::lua_interpreter::display(gui2::dialogs::lua_interpreter::APP);
 }
 
+static void make_screenshot()
+{
+	const surface& screenshot = make_neutral_surface(CVideo::get_singleton().getSurface());
```

Comment:
I'd prefer obtaining reference to the window with `get_window()->video()` instead of `CVideo::get_singleton()`.

Also, just get your own `surface` object instead of using a const reference.